platform-sdk/swirlds-unit-tests/structures/swirlds-merkle-test/build.gradle.kts

```diff
@@ -15,25 +15,26 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
 }
 
 dependencies {
-  // Individual Dependencies
-  implementation(project(":swirlds-merkle"))
-  compileOnly(libs.spotbugs.annotations)
+    // Individual Dependencies
+    implementation(project(":swirlds-merkle"))
```

Comment:
intentional?

Reply:
I assume that files were never touched after we introduced the new formatting and spotless.